data/json/recipes/recipe_ammo.json

```diff
@@ -1,685 +1,565 @@
+  "result" : "arrow_wood_unfletched",
   "category" : "CC_AMMO",
   "subcategory": "CSC_AMMO_ARROWS",
   "skill_used" : "fabrication",
-  "skills_required": [[ "archery", 2 ], [ "survival", 1 ]],
-  "difficulty" : 3,
-  "time" : 11000,
+  "difficulty" : 1,
+  "time" : 8000,
   "autolearn" : true,
-  "book_learn" : [[ "recipe_arrows", 1 ] , [ "book_archery", 3]],
   "qualities":[
-    {"id":"CUT","level":1},
-    {"id":"HAMMER","level":1}
+    {"id":"CUT","level":1}
   ],
   "components" : [
     [
-      [ "superglue", 1 ],
-      [ "bone_glue", 1 ],
-      [ "thread", 20 ],
-      [ "sinew", 20 ],
-      [ "plant_fibre", 20 ]
-    ],
-    [
-      [ "stick", 1 ],
-      [ "broom", 1 ],
-      [ "2x4", 1 ],
-      [ "bee_sting", 1 ]
-    ],
-    [
-      [ "bone", 5 ],
-      [ "bone_human", 5 ],
-      [ "bone_tainted", 10 ],
-      [ "rock", 1 ],
-      [ "ceramic_shard", 1 ],
-      [ "scrap", 2 ],
-      [ "nail", 10 ],
-      [ "sharp_rock", 1 ]
-    ],
-    [
-      [ "feather", 10 ],
-      [ "duct_tape", 40 ],
-      [ "paper", 40 ],
-      [ "bag_plastic", 1 ],
-      [ "bottle_plastic_small", 1 ],
-      [ "box_small", 1 ],
-      [ "plastic_chunk", 1 ],
-      [ "aluminum_foil", 40 ],
-      [ "cash_card", 2 ]
+      [ "stick", 10],
+      [ "broom", 5 ],
+      [ "2x4", 10 ]
     ]
   ]
-},{
+},
+{
   "type" : "recipe",
-  "result" : "arrow_small_game_fletched",
+  "result" : "arrow_wood",
   "category" : "CC_AMMO",
   "subcategory": "CSC_AMMO_ARROWS",
   "skill_used" : "fabrication",
-  "skills_required": [ "survival", 1 ],
-  "difficulty" : 1,
-  "time" : 11000,
+  "difficulty" : 2,
+  "time" : 10000,
   "autolearn" : true,
-  "book_learn" : [[ "recipe_arrows", 1 ] , [ "pocket_survival", 1], [ "book_archery", 2]],
   "qualities":[
     {"id":"CUT","level":1}
   ],
   "components" : [
     [
-      [ "superglue", 1 ],
-      [ "bone_glue", 1 ],
-      [ "thread", 20 ],
-      [ "sinew", 20 ],
-      [ "plant_fibre", 20 ]
-    ],
-    [
-      [ "stick", 1 ],
-      [ "broom", 1 ],
-      [ "2x4", 1 ],
-      [ "bee_sting", 1 ]
-    ],
-    [
-      [ "bone", 5 ],
-      [ "bone_human", 5 ],
-      [ "bone_tainted", 10 ],
-      [ "rock", 1 ],
-      [ "ceramic_shard", 1 ],
-      [ "scrap", 2 ],
-      [ "nail", 10 ],
-      [ "sharp_rock", 1 ]
+      [ "stick", 10],
+      [ "broom", 5 ],
+      [ "2x4", 10 ]
     ],
     [
-      [ "feather", 10 ],
-      [ "duct_tape", 40 ],
-      [ "paper", 40 ],
-      [ "bag_plastic", 1 ],
-      [ "bottle_plastic_small", 1 ],
+      [ "feather", 7 ],
+      [ "duct_tape", 30 ],
+      [ "paper", 30 ],
+      [ "bottle_plastic_small", 4 ],
+	  [ "bottle_plastic", 2 ],
+	  [ "bowl_plastic", 3 ],
+	  [ "plastic_shopping_bag", 2],
+	  [ "cup_plastic", 3 ],
+	  [ "jug_plastic", 2],
       [ "box_small", 1 ],
-      [ "plastic_chunk", 1 ],
-      [ "aluminum_foil", 40 ],
+      [ "plastic_chunk", 2 ],
+      [ "aluminum_foil", 30 ],
       [ "cash_card", 2 ]
```

Comment:
Like how 'cording' deals with various strings and such. Still, given these will probably only be used in this configuration for archery-related, don't know if it's very needed. But will do it if required.